ftl/common/single_layer_image.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This package defines the shared cli args for ftl binaries."""
+
+import abc
+
+from containerregistry.client.v2_2 import docker_digest
+
+_DEFAULT_TTL_WEEKS = 1
+
+
+class BaseLayerImage(object):
```

Comment:
Maybe name this just "Layer" instead of LayerImage. It has a reference to an image, but it isn't an image itself.

Reply:
done